### PR TITLE
Math magicians: Add tests

### PR DESCRIPTION
**Pull request by ernestmusong**
## We have added the following tests: 

**Components tests**

- Layout component
- Home component
- Navbar component
- Calculator component
- Quote componet
- Error component

**Files**

- calculate.js
- operate.js

### CODE REVIEW

File: src/tests/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+describe('Home', () => {
+  test('should render the home component', () => {
+    const tree = renderer.create(<Home />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```
Comment (nahnahdev) [inline src/tests/Home.test.js]: - [ ] Great job so far Team :muscle: , you did an amazing job writing all the tests for your components. However, this particular snapshot test failed when I run the tests suite. So kindly fix this so all your tests can pass successfully :+1: 
- I have attached a screenshot reference below :100: 
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @ernestmusong  and @theodoku.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- `operate.js `and `calculate.js` modules testes using Jest :heavy_check_mark: 
- Usage of Jest snapshots for testing all the components :heavy_check_mark: 
- Usage of React Testing Library to simulate user interaction :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [ OPTIONAL ] I will suggest you include your partner's information in the `Authors section` of the  README since you worked on this milestone together :100:  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Home.test.js (hunk 1 above)
Comment (ernestmusong) [reply src/tests/Home.test.js]: > * [ ]  Great job so far Team 💪 , you did an amazing job writing all the tests for your components. However, this particular snapshot test failed when I run the tests suite. So kindly fix this so all your tests can pass successfully 👍
> * I have attached a screenshot reference below 💯
>   image

Thanks @NahnahAJ for your feedback, I have now fixed the test and all passing.

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @ernestmusong  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- All tests passing :heavy_check_mark: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**